draft_python/build.py

```diff
@@ -0,0 +1,30 @@
+import sys
```

Comment:
So this is taking a draft run and converting it to a viewable HTML page, right? Could we name it something a bit clearer? Maybe like 'generateDeckView' or something.

Reply:
For better or for worse, python naming heavily favors shorter names: https://www.python.org/dev/peps/pep-0008/#package-and-module-names

This script actually reads the draft log, runs the deckbuilder on each of the pools, and THEN outputs the decks as HTML. So it'd have to be deckbuild_and_generate_view.py or something to be (more) fully descriptive. 

In general I do like the python style since imo there are diminishing returns on longer names. How about "deckbuild.py" as a compromise?

Reply:
Sure, I'm good with `deckbuild.py`. Build is just super generic.